app/main/helpers/frameworks.py

```diff
@@ -38,6 +40,18 @@ def get_framework_lot(framework, lot_slug):
         abort(404)
 
 
+def order_frameworks_for_reuse(frameworks):
+    """Sort frameworks by reuse suitability.
+
+    If a declaration has the reuse flag set and is the most recently closed framework then that's our framework.
+    """
+    return sorted(
+        filter(lambda i: i['allow_declaration_reuse'] and i['application_close_date'], frameworks),
+        key=lambda i: datetime.strptime(i['application_close_date'], DATETIME_FORMAT),
```

Comment:
Why do we need to string format the date here?  Can't we just order by the datetime values themselves?